library/Icingadb/Widget/ItemList/BaseHostListItem.php

```diff
@@ -23,6 +24,11 @@ abstract class BaseHostListItem extends StateListItem
 {
     use NoSubjectLink;
 
+    /**
+     * Create new subject link
+     *
+     * @return BaseHtmlElement
```

Comment:
Cant' this be a return type declaration as well?

Reply:
Nope